amp-ad type weborama-display

bob-v-t opened the pull request:
Hi guys,

I hereby submit the pull request mentioned in issue number #2832. ([backup link](https://github.com/ampproject/amphtml/issues/2832))

Please let me know if there is anything I can do to smoothen the integration process!

Thanks,
Bob

bob-v-t commented:
I signed it!

I've signed as an employee of the company Weborama.

cramforce commented on ads/weborama.md:
```diff
@@ -0,0 +1,95 @@
+limitations under the License.
+-->
+
+# Weborama
```
Please add a link to your own docs.

bob-v-t replied:
Unfortunately, I think that we only have internal documentation that explains our tags. We usually configure the tags for our clients ourselves before sending them out.

I didn't see links to documentation in most of the other ad type definitions. Is it OK if we skip this step too?

Or were you referring to the copyright banner at the top of the file?

bob-v-t replied:
@cramforce My Google CLA (Contributor License Agreement) was voided.. is that going to be a problem, or won't this step be necessary for now?

cramforce replied:
On the original question: In that case just add a link to your contact form, help center or whatever.

I'll take a look at the CLA.

bob-v-t replied:
Ok, will do so tomorrow when I'm behind my PC.

Thanks for looking into this!

cramforce commented:
Looks good. 2 small comments.

cramforce commented:
For the CLA: Did you submit a corporate or individual CLA?

bob-v-t commented:
About the CLA: It was a coorporate request.

cramforce commented:
@bob-v-t Great. The corporate CLAs can take a few business days to process. Personal is instant.

bob-v-t commented:
@cramforce I rebased and added the email and phone of our support this morning. I also just signed the CLA again, since the previous one was rejected. Let's hope this one works out :+1:

bob-v-t commented:
Thanks @cramforce!
